sdk/core/azure-core/azure/core/pipeline/transport/_base.py

```diff
@@ -54,6 +54,8 @@
 
 from http.client import HTTPResponse as _HTTPResponse
 
+from black import FileContent
```

Comment:
This isn't a dependency?

Reply:
yep removing it rn, intellisense added it incorrectly

Reply:
Oh maybe this is VSCode auto-importing?

Reply:
yeah it just added when I tabbed and I didn't notice it, fixed now though, thanks!